packages/lit-dev-content/src/icons/check-circle-crossed-icon.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {html} from 'lit';
+
+// Source: https://fonts.google.com/icons?selected=Material+Icons+Outlined:unpublished
```

Comment:
URL seems wrong

Reply:
it works for me, what do you see? I get this:

![image](https://user-images.githubusercontent.com/5981958/164109941-22941ca6-6582-4cc7-af21-f5c40a0f0255.png)

Reply:
Sorry my mistake, you're right it works fine.